test/TesterAzureUtils/Persistence/PersistenceGrainTests_AzureTableStore.cs

```diff
@@ -41,7 +41,7 @@ protected override void ConfigureTestCluster(TestClusterBuilder builder)
             {
                 Guid serviceId = Guid.NewGuid();
                 builder.Options.InitialSilosCount = 4;
-
+                builder.Options.UseTestClusterMembership = false;
```

Comment:
Aren't other external membership tests (non azure) affected by the same issue. They wouldn't show up as we don't run them in CI